src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -385,4 +386,19 @@ public Job call() throws SaltStackException {
         };
         return executor.submit(callable);
     }
+
+  public Stats stats() throws SaltStackException {
```

Comment:
I agree with @lucidd here. @isbm why do you think a synchronous method is superfluous?